shared/actions/json/waiting.json

```diff
@@ -8,6 +7,9 @@
       "key": ["string", "Array<string>"],
       "error?": "RPCError"
     },
+    "batchChangeWaiting": {
+      "changes": "Array<{key: string | Array<string>, increment: boolean, error?: RPCError}>"
+    },
```

Comment:
also what about incrementWaiting / decrementWaiting? Do we care about batching those too?

Reply:
unclear. honestly most of these should hopefully go away by letting the engine do it. lemme add a pass of that cleanup in this pr

Reply:
ok i remove a ton of manual increment in teams. we do explicitly increment / decrement but its only in 2 places really and not a perf matter